src/infrastructure/api/index.ts

```diff
@@ -20,10 +22,8 @@ const { API_VERSION } = process.env;
  * next()
  * })
  */
-app.use((req, __, next) => {
+app.use((_, __, next) => {
```

Comment:
If you're doing this so TS shuts up, usually you can also name your variables something descriptive prefixed by an underscore e.g. `_request`